Lombiq.OrchardCoreApiClient/ApiClient.cs

```diff
@@ -31,7 +32,7 @@ public class ApiClient : IDisposable
             };
 #pragma warning restore CA5399
 
-            return RestService.For<IOrchardCoreApi>(_httpClient);
+            return RefitHelper.WithNewtonsoft<IOrchardCoreApi>(_httpClient);
```

Comment:
Why do we need to use Json.NET? Add a note about it here.

Reply:
Mainly because OC uses Json.NET so there will be slight implementation differences that explode the code if we don't do that. When I tried to use STJ I got an exception saying something like it can't convert string to int.